focal/4_gs1_identification_examples.html

```diff
@@ -0,0 +1,94 @@
+<section>
+      <h3>Focal Use Case: Chain of GS1 Credentials to Identify a Trade Item</h3>
+
+      <p>
+            These examples are based on the document <a href="https://ref.gs1.org/gs1/vc/data-model/" rel="nofollow">"GS1 Verifiable Credentials - Data Model and Validations"</a>. The examples use Verifiable Credentials Data Model 2.0.
```

Comment:
Have you validated these credentials in the JSON-LD playground or the VC test suite?

At cursory glance, they look fine, but its worth doing the validation step before we publish.

I'll add an issue to do the same with the other example, which may no longer be conformant with VC2.0.

Reply:
The examples are intended only to illustrate business rules and are not intended to be structurally complete (e.g., no proof section). So no, they haven't and won't be validated for structural correctness.